storage/innobase/fts/fts0que.cc

```diff
@@ -953,6 +953,18 @@ fts_query_free_doc_ids(
 	query->total_size -= SIZEOF_RBT_CREATE;
 }
 
+
+/*******************************************************************//**
```

Comment:
New InnoDB code should use the new InnoDB function header comment style (see e.g. fts_ast_string_create, innobase_mysql_tmpfile, fil_rename_tablespace_check for examples)